src/instructlab/configuration.py

```diff
@@ -453,6 +470,11 @@ def get_default_config() -> Config:
             deepspeed_cpu_offload_optimizer=False,
             additional_args={},
             is_padding_free=False,
+            phased_phase1_num_epochs=10,
+            phased_phase1_samples_per_save=25000,
+            phased_phase2_num_epochs=10,
+            phased_phase2_samples_per_save=25000,
+            phased_mt_bench_judge=DEFAULTS.JUDGE_MODEL_MT,
```

Comment:
Shouldn't this be updated the same as: https://github.com/instructlab/instructlab/pull/1957

Reply:
Yes, artifact from before your change got merged!